enso/config.py

```diff
@@ -14,15 +14,43 @@
 FEATURES_DIRECTORY = "Features"
 
```

Comment:
We should think about how we handle tracking this file in the future.  Feels like this is something we shouldn't be committing.